include/leveldb/c.h

```diff
@@ -189,6 +189,8 @@ LEVELDB_EXPORT void leveldb_options_set_block_restart_interval(
     leveldb_options_t*, int);
 LEVELDB_EXPORT void leveldb_options_set_max_file_size(leveldb_options_t*,
                                                       size_t);
+LEVELDB_EXPORT void leveldb_options_set_disable_seek_autocompaction(leveldb_options_t*,
+	                                                                uint8_t);
```

Comment:
extreme nit: inconsistency between using spaces and tabs

Reply:
This has been addressed 👍